### PR TITLE
Explain what this is and how to set it up

### PR DESCRIPTION
**Pull request by gabebw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
+
+Q: No seriously, what is it?
+A: Way back in the 1990s, there were no search engines, but people still wanted
+   that sweet sweet traffic. And thus the webring: a group (or ring) of websites
+   with links to other websites in the ring.
 
-After you have cloned this repo, run this setup script to set up your machine
-with the necessary dependencies to run and test this app:
+Q: So it's some weird old thing?
+A: Um it's [vintage, and so adorable](https://www.youtube.com/watch?v=kfLSjobM9bg).
```
Comment (adarsh) [inline README.md]: Yisssssss.

File: README.md [hunk 2]
```diff
@@ -1,20 +1,31 @@
...
 
-    % ./bin/setup
+Q: How do I get in on this?
+A: Make up an identifier for yourself like `gabebw`. Then add links back and
+   forward in the ring: `URL.com/gabebw/previous` and `URL.com/gabebw/next`.
+   You'll be automatically added to the ring.
```
Comment (adarsh) [inline README.md]: Can we provide arrow PNGs for the truly lazy?

File: README.md [hunk 3]
```diff
@@ -1,20 +1,31 @@
...
 
-After you have cloned this repo, run this setup script to set up your machine
-with the necessary dependencies to run and test this app:
+Q: So it's some weird old thing?
+A: Um it's [vintage, and so adorable](https://www.youtube.com/watch?v=kfLSjobM9bg).
 
-    % ./bin/setup
+Q: How do I get in on this?
+A: Make up an identifier for yourself like `gabebw`. Then add links back and
+   forward in the ring: `URL.com/gabebw/previous` and `URL.com/gabebw/next`.
```
Comment (adarsh) [inline README.md]: Should you use the real URL here?

**Comment by adarsh** [comment]
:sheep:

File: README.md (hunk 3 above)
Comment (gabebw) [inline README.md]: I need to buy it first!

File: README.md (hunk 2 above)
Comment (gabebw) [inline README.md]: The truly lazy should use HTML entities: 3ffe3746b8fae8e1e5a9cb461044aa4babefca10

**Comment by gabebw** [comment]
@adarsh, as a contributor who read the branch name, do you feel better about contributing?

**Comment by gabebw** [comment]
https://twitter.com/adarshp/status/663814355536908288

File: README.md (hunk 3 above)
Comment (adarsh) [inline README.md]: image
https://www.uniteddomains.com/domain/searchresult/?domain=hotlinewebring&multi-tld=-top20-

**Comment by adarsh** [comment]
I feel marvelous about everything happening here. Would comment again, five stars.

**Pull request merged**